doc/locale/ja/LC_MESSAGES/install/cmake_options.po

```diff
@@ -0,0 +1,161 @@
+"Files (x86)/groonga` です。"
+
+msgid "Here is an example of specifying ``C:\\Groonga`` as an install folder::"
+msgstr "以下は ``C:\\Groonga`` をインストール先に指定する例です。::"
+
+msgid "``-DGRN_WITH_MRUBY``"
+msgstr ""
+
+msgid "Enables mruby support."
+msgstr "mrubyサポートを有効化します。"
+
+msgid ""
+"You can use the :doc:`/reference/sharding` plugin and :doc:`/reference/"
+"commands/ruby_eval` with the mruby support enabled."
+msgstr ""
+"mrubyサポートを有効にすることで :doc:`/reference/sharding` プラグインと :doc:"
+"`/reference/commands/ruby_eval` が使用可能になります。"
+
+msgid "The default is ``off``."
+msgstr "デフォルトは ``off`` です。"
+
+msgid ""
+"Groonga builds bundled mruby if the mruby support is enabled. In order to "
+"build mruby, you must install some requierd libraries. See the `mruby "
+"compile guide <https://github.com/mruby/mruby/blob/master/doc/guides/compile."
+"md>`_ for more details."
+msgstr ""
+"mrubyサポートが有効な場合、Groongaは同梱しているmrubyをビルドします。そのた"
+"め、mrubyをビルドするのに必要なライブラリをインストールする必要があります。詳"
+"細は `mrubyのコンパイルガイド <https://github.com/mruby/mruby/blob/master/"
+"doc/guides/compile.md>`_ を参照してください。"
+
+msgid "Here is an example of enabling the mruby support::"
+msgstr "以下はmrubyサポートを有効にする例です::"
+
+msgid "``-DGRN_WITH_APACHE_ARROW``"
+msgstr ""
+
+msgid "Enables Apache Arrow support."
+msgstr "Apache Arrowサポートを有効化します。"
+
+msgid ""
+"In addition to using Apache Arrow IPC streaming format output, you can also "
+"use multithreading processing that is used in :ref:`select-n-workers` and :"
+"doc:`/reference/functions/query_parallel_or` with the Apache Arrow support."
+msgstr ""
+"Apache Arrowサポートを有効化すると、Apache ArrowのIPC streaming formatが使え"
+"るだけでなく、 :ref:`select-n-workers` や :doc:`/reference/functions/"
+"query_parallel_or` でマルチスレッドで処理できるようになります。"
+
+msgid ""
+"You can install Apache Arrow following to `the official installation "
+"procedure <https://arrow.apache.org/install/>`_."
+msgstr ""
+"Apache Arrowは `公式のドキュメントの手順 <https://arrow.apache.org/install/"
+">`_ にしたがってインストールしてください。"
+
+msgid "Here is an example of enabling the Apache Arrow support::"
+msgstr "以下はApache Arrowサポートを有効にする例です::"
+
+msgid ""
+"If you install Apache Arrow manually, you need to use the :ref:`windows-"
+"cmake-cmake-prefix-path` option."
+msgstr ""
+"Apache Arrowを手動でインストールしている場合、 :ref:`windows-cmake-cmake-"
+"prefix-path` オプションを指定してください。"
+
+msgid "``-DCMAKE_PREFIX_PATH=PATHS``"
+msgstr ""
+
+msgid "Adds search paths of ``.cmake`` files."
+msgstr "``.cmake`` ファイルの検索パスを追加します。"
+
+msgid "You can specify multiple paths using ``:`` to separate."
+msgstr "``:`` で区切ることで複数のパスを指定することができます。"
+
+msgid ""
+"When using libraries installed via a package manager, you do not need to "
+"specify this parameter because ``.cmake`` files for the libraries are in the "
+"default search paths of CMake."
+msgstr ""
+"パッケージ管理システムを使用してインストールしたライブラリを使いたい場合は、 "
+"CMake のデフォルトの検索パスに ``.cmake`` ファイルが存在するためこのパラメー"
+"タを指定する必要はありません。"
+
+msgid ""
+"When using manual built libraries, you need to specify ``.cmake`` file paths "
```

Comment:
libraries built by yourself?

"libraries installed not in system directories such as `/usr`" or something may be better. "Manual build" isn't related to this. "install location" is related.

Reply:
Fixed.